packages/common/infra/src/modules/workspace/metadata.ts

```diff
@@ -1,3 +1,7 @@
 import type { WorkspaceFlavour } from '@affine/env/workspace';
 
-export type WorkspaceMetadata = { id: string; flavour: WorkspaceFlavour };
+export type WorkspaceMetadata = {
+  id: string;
+  flavour: WorkspaceFlavour;
+  initialized: boolean;
```

Comment:
```suggestion
  initialized?: boolean;
```

Make this nullable, Workspace metadata may be stored in the cache, and the metadata in the cache does not have this